tempesta_fw/t/functional/testers/stress.py

```diff
@@ -10,7 +10,7 @@ class StressTest(unittest.TestCase):
     """ Test Suite to use HTTP benchmarks as a clients. Can be used for
     functional testing of schedulers and stress testing for other components.
     """
-
+    bce = 0
```

Comment:
this apply to only one test, so shouldn't be listed in the base test class. Instead add to derived class if required.

Reply:
no, bce used in assert_tempesta()

Reply:
and assert_clients()